packages/flutter_driver/test/src/real_tests/timeline_summary_test.dart

```diff
@@ -869,9 +871,11 @@ void main() {
         final List<Map<String, dynamic>> events = <Map<String, dynamic>>[];
         const int num30Hz = 10;
         const int num60Hz = 20;
+        const int num80Hz = 20;
         const int num90Hz = 20;
         const int num120Hz = 40;
         const int numIllegal = 10;
+        const int totalFrames = num30Hz + num60Hz + num80Hz + num90Hz + num120Hz + numIllegal;
 
         // Add 30hz frames
         events.addAll(_populateEvents(numberOfEvents: num30Hz,
```

Comment:
Can you add some tests here where these fields are null to make sure it doesn't crash?

Reply:
Did you mean to test the perf_test code above for when data['30hz_frame_percentage'] is null? 
I couldn't find any existing test to test the PerfTest class. Do we have a way to test it?

Reply:
I mean a timeline summary test for when the 30hz_frame_percentage is null

Reply:
summary.percentageOf30HzFrames is non-null. 
Did you mean the event of 30Hz frame is null? I think that is also covered in previous tests like `Identify illegal refresh rates`

Reply:
I just mean a test that would fail on presubmit without your change, so we could have caught it last time before landing in devicelab

Reply:
@dnfield Yep, that would probably going to be a test for perf_test. Since the crash is an explicit throw in perf_test where data[<key>] is null. I can look into it,

Reply:
@dnfield added tests, PTAL